Mathmagician/Commands/Even.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician.Commands
+{
+    class Even : Command
+    {
+        public override void CreateNumbers(int quantityOfNumsToCreate)
+        {
+            for (var i = 0; Numbers.Count != quantityOfNumsToCreate; i++)
+            {
+

```

Comment:
You're missing something here. Right now, this method just counts, but doesn't affect the list in Even.